lib/wda/xcodebuild.js

```diff
@@ -152,7 +154,6 @@ class XcodeBuild {
     // first do a build phase
```

Comment:
the previous condition can be deleted, since Xcode 7 is not supported anyway

Reply:
That should be part of a different PR.